Rivet/Functions/New-Migration.ps1

```diff
@@ -38,6 +41,15 @@ function New-Migration
         $migrationPath = [IO.Path]::GetFullPath( $migrationPath )
         New-Item -Path $migrationPath -Force -ItemType File
 
+        $schemaPs1Path = Join-Path -Path $Path -ChildPath $script:schemaMigrationId
+        if (-not (Test-Path -Path $schemaPs1Path))
+        {
+            $script:schemaPreamble | Set-Content -Path $schemaPs1Path
+            $msg = "Rivet created ""$($schemaPs1Path | Resolve-Path -Relative)"", a file where Rivet to stores a " +

```

Comment:
Extra "to" in the warning message. Change "a" to "the", i.e. "stores the database's".